src/types/device/device.ts

```diff
@@ -50,4 +50,12 @@ export interface DeviceList extends DeviceBase {
   id: string;
 }
 
+export interface DeviceEncounters {
```

Comment:
a record represents single object right.

```suggestion
export interface DeviceEncounter {
```

Reply:
yes it does 🙂‍↕️, made the change as requested.